estimagic/tests/visualization/test_comparison_plot.py

```diff
@@ -0,0 +1,404 @@
+import json
+
+import numpy as np
+import pandas as pd
+import pytest
+from pandas.testing import assert_frame_equal
```

Comment:
numpy does ``import numpy.testing as npt``. Maybe you can use ``import pandas.testing as pdt`` which is more explicit.

Reply:
Adopted in 0846b2d5aa68a1880f930eade981bce6930216aa.